jquery.facetview.js

```diff
@@ -1581,6 +1582,11 @@ search box - the end user will not know they are happening.
                     $(options.searchbox_class).bindWithDelay('keyup',dosearch,options.freetext_submit_delay);
                 }
 
+                // Properly set size for each facet
+                $.each(options.facets, function(facet){
+                  options.facets[facet]["size"] = options.facet_size;
```

Comment:
I trust that you know what you're doing.  Let's chat if I can be helpful to you.

As for git blame, you might have to do some gymnastics, since the fork to add solr back in lost most of the OKFN history.  I would recommend cloning that repo to your hard disk for comparison: https://github.com/okfn/facetview